snap/snapcraft.yaml

```diff
@@ -27,6 +27,22 @@ slots:
     source:
       write:
         - "$SNAP_COMMON/run/switch"
+  ovn:
+    interface: content
+    source:
+      write:
+        - "$SNAP_COMMON/run/ovn"
+  ovn-central:
```

Comment:
if `ovn` gets renamed to `ovn-central`, this could then be named `ovn-central-data`.